Cue/app/common/SyncConflict.js

```diff
@@ -0,0 +1,169 @@
+// @flow
+
+import React from 'react'
+import { View, Text, ListView, Dimensions, Navigator, Platform, Alert} from 'react-native'
+import { Deck } from '../api/types';
+import { connect } from 'react-redux'
+import CueIcons from './CueIcons'
+import CueHeader from './CueHeader'
+import { resolveConflict } from '../actions/library'
+import { SegmentedControls } from 'react-native-radio-buttons'
+
+  //TODO: style this page
+
+type Props = {
+  failedSyncs: Array<Deck>,
+  navigator: Navigator,
+
+  // Redux
+  localDecks: Array<Deck>
+}
+
+const styles = {
+  container: {
+    flex: 1,
+    backgroundColor: 'white',
+  },
+}
+
+class SyncConflict extends React.Component {
+  props: Props
+
+  state: {
+    dataSource: ListView.dataSource,
+    deviceOrientation: string,
```

Comment:
Is the `deviceOrientation` state used for anything? In `LibraryListView` it's used so that the `ListView` is forced to unmount and re-mount so that the `DeckThumbnail`s are sized correctly, but I'm not sure I see why it's used here.